build.sbt

```diff
@@ -308,6 +308,7 @@ val newtypeVersion              = "0.4.4"
 val pprintVersion               = "0.5.9"
 val pureconfigVersion           = "0.13.0"
 val refinedVersion              = "0.9.14"
+val rustVersion                 = "1.40.0-nightly"
```

Comment:
We need to make sure this is the specific nightly version we use too (b520af6fd).